flow-server/src/main/java/com/vaadin/flow/server/BootstrapHandler.java

```diff
@@ -1563,9 +1563,23 @@ protected static ObjectNode getStatsJson(DeploymentConfiguration config)
         String statsJson = DevBundleUtils.findBundleStatsJson(
                 config.getProjectFolder(), config.getBuildFolder());
         Objects.requireNonNull(statsJson,
-                "Frontend development bundle is expected to be in the project"
-                        + " or on the classpath, but not found."
-                        + " Add 'com.vaadin.vaadin-dev-server' dependency or include it transitively via 'com.vaadin.vaadin-dev' to let Vaadin build the development bundle automatically.");
+                """
+                        Frontend development bundle is expected to be in the project or on the classpath, but not found.
+                        Add 'com.vaadin.vaadin-dev' dependency to let Vaadin re-use the pre-compiled development bundle
```

Comment:
```suggestion
                        Add 'com.vaadin:vaadin-dev' dependency to let Vaadin re-use the pre-compiled development bundle
```

The "gradle syntax" as shorthand for dependency is quite common (even among Maven users), only dots I have rarely seen.